src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -66,8 +66,15 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     case SubmitWorkflow(wdlSource, wdlJson, inputs) =>
       submitWorkflow(wdlSource, wdlJson, inputs, maybeWorkflowId = None) pipeTo sender
     case WorkflowStatus(id) => dataAccess.getWorkflowState(id) pipeTo sender
-      // TODO actually attempt to abort workflow
-    case WorkflowAbort(id) => sender ! Option(WorkflowAborted)
+    case WorkflowAbort(id) =>
+      val workflowActorOption = workflowStore.toMap.get(id)
+      workflowActorOption match {
+        case Some(x) =>
+          x ! WorkflowAbort(id)
+          sender ! Some(WorkflowAborted)
```

Comment:
Shouldn't this send `WorkflowAborting`?  We don't know the workflow is aborted yet.